docs/samples/browser-plugin-meetings/index.html

```diff
@@ -104,13 +105,123 @@ <h2 class="collapsible">Meeting Management</h2>
 
               <form id="oauth" class="hidden">
                 <div>
-                  <button id="oauth-login-btn" type="submit">Login</button>
+                  <button id="oauth-login-btn" type="submit" class="btn-code">Login</button>
                   <p id="oauth-status" class="status-par">Not Logged In.</p>
                 </div>
               </form>
             </fieldset>
           </div>
 
+          <!-- Streams / effects -->
```

Comment:
there is no UI to allow unpublishing of audio or video tracks.

Reply:
Yes, we can add it. This can be done in a separate Jira which I can create.

But, when you say that we need to allow unpublishing of audio/video streams then in that case we will also need a functionality to publish them again in the same meeting, correct?

Reply:
yeah

Reply:
Jira: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-473915